_i18n/es.yml

```diff
@@ -145,12 +145,12 @@ security_page:
       los usuarios tengan conocimiento de que su privacidad está siendo protegida
       de la manera que diseñamos la plataforma.'
     p_2: 'Para obtener más información, visite el sitio web de login.gov [Centro de
-      ayuda](site.baseurl /help) o [contáctenos](site.baseurl / contact). También
-      puede visitar nuestro [repositorio de código abierto](https: / /github.com/18F/identity-idp){:target="_blank"}.'
+      ayuda]({{site.baseurl}}/help) o [contáctenos]({{site.baseurl}}/contact). También
```

Comment:
Oh whoops, I forgot about the `replace: 'site.baseurl', site.baseurl | markdownify` bit